src/main/java/seedu/address/model/task/DateTimeDue.java

```diff
@@ -58,6 +58,11 @@ public DateTimeDue(Optional<String> optionalDueAt) {
                             .parseFirstMatching(trimmedDueAt, LocalDateTime::from, ALLOWED_DATETIME_FORMATS));
                 })
                 .orElse(Optional.empty());
+        if (optionalDueAt.isEmpty()) {
```

Comment:
do you want to consider moving these few lines of code into the map function from line 55? you can just write a one liner `val = trimmedDueAt` and i think it should do the trick?

Reply:
Okay